02.general/02.how-to-create-your-own-ad-filters/docs.en.md

```diff
@@ -880,6 +880,14 @@ CSS 3.0 is not always enough to block ads. To solve this problem AdGuard extends
 > #### Application area
 > Extended selectors can be used in any cosmetic rule, whether they are [element hiding rules](#cosmetic-elemhide-rules) or [CSS rules](#cosmetic-css-rules).
 
+> #### Syntax
+>  The syntax for extended CSS rules:
```

Comment:
It'd be better to reword it a little bit, add an introduction.

Smth like this:

Regardless of the CSS pseudo-classes you are using in the rule, you can use special markers to make these rules use the "Extended CSS" engine. It is recommended to use these markers for all "extended CSS" cosmetic rules so that it was easier to find them.